From e9e2386e93dac604242fb369c92c8110e9a6fd8f Mon Sep 17 00:00:00 2001
From: Radhey Shyam Pandey <radhey.shyam.pandey@xilinx.com>
Date: Mon, 8 Jun 2020 19:46:29 +0530
Subject: [PATCH 1373/1852] dmaengine: xilinx: add mcdma reference in
 axidmatest documentation

commit 3309c34f0f2f91de05b9862ff19c93625dcfa315 from
https://github.com/Xilinx/linux-xlnx.git

Mention mcdma as a supported configuration in axidmatest client Kconfig
description and source file. No functional change.

Signed-off-by: Radhey Shyam Pandey <radhey.shyam.pandey@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/dma/Kconfig             | 2 +-
 drivers/dma/xilinx/axidmatest.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/dma/Kconfig b/drivers/dma/Kconfig
index 3a0df262e0d0..efee07b3ff2d 100644
--- a/drivers/dma/Kconfig
+++ b/drivers/dma/Kconfig
@@ -717,7 +717,7 @@ config DMA_ENGINE_RAID
 	bool
 
 config XILINX_DMATEST
-	tristate "DMA Test client for AXI DMA"
+	tristate "DMA Test client for AXI DMA and MCDMA"
 	depends on XILINX_DMA
 	help
 	  Simple DMA test client. Say N unless you're debugging a
diff --git a/drivers/dma/xilinx/axidmatest.c b/drivers/dma/xilinx/axidmatest.c
index d7658ae90896..35718fc44ee2 100644
--- a/drivers/dma/xilinx/axidmatest.c
+++ b/drivers/dma/xilinx/axidmatest.c
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: GPL-2.0-or-later
 /*
- * XILINX AXI DMA Engine test module
+ * XILINX AXI DMA and MCDMA Engine test module
  *
  * Copyright (C) 2010 Xilinx, Inc. All rights reserved.
  *
-- 
2.31.1

